apiv2/client.go

```diff
@@ -130,6 +130,23 @@ func NewRESTClientForHost(u, username, password string, opts *config.Options) (*
 	return NewRESTClient(v2SwaggerClient, opts, authInfo), nil
 }
 
+// NewRESTClientWithAuthFunc constructs a new REST client containing a swagger API client using the defined
+// host string and basePath, the additional Harbor v2 API suffix as well as customer auth func,eg: basic auth or token auth.
```

Comment:
I'm sure you meant `custom` instead of `customer` here?
```suggestion
// host string and basePath, the additional Harbor v2 API suffix as well as a custom auth func, e.g. basic auth or token auth.
```